site/docs/use-cases/speech-processing/_sections/_run_model/index.mdx

```diff
@@ -0,0 +1,41 @@
+import CodeExampleCPP from './_code_example_cpp.mdx';
+import CodeExamplePython from './_code_example_python.mdx';
+
+## Run Model Using OpenVINO GenAI
+
+OpenVINO GenAI introduces the [`WhisperPipeline`](https://docs.openvino.ai/2025/api/genai_api/_autosummary/openvino_genai.WhisperPipeline.html) pipeline for inference of speech processing Whisper models.
+You can construct it straight away from the folder with the converted model.
+It will automatically load the model, tokenizer, detokenizer and default generation configuration.
+
+:::info
+`WhisperPipeline` expects normalized audio files in WAV format at sampling rate of 16 kHz as input.
+:::
+
+<LanguageTabs>
+    <TabItemPython>
+        <Tabs groupId="device">
+            <TabItem label="CPU" value="cpu">
+                <CodeExamplePython device="CPU" />
+            </TabItem>
+            <TabItem label="GPU" value="gpu">
+                <CodeExamplePython device="GPU" />
+            </TabItem>
+        </Tabs>
+    </TabItemPython>
+    <TabItemCpp>
```

Comment:
Cpp sample requires utils from repository, like `utils::audio::read_wav`. Do we need to highlight it additionally?
I guess same applies for image and VLM pipelines.